ci/gpu/build.sh

```diff
@@ -56,7 +56,8 @@ conda install -c pytorch \
     "cmake" \
     "cython" \
     "pytest" \
-    "s3fs"
+    "s3fs" \
+    "ipython"
```

Comment:
Why are changes to our build process being made in a notebook PR?

Reply:
the gpu build script adds all the dependencies for our unit tests and notebooks. gpu build was failing because ipython was not installed and causing error in this notebook.